packages/addon-modernjs/src/preset.ts

```diff
@@ -0,0 +1,58 @@
+import type { AppNormalizedConfig, AppTools } from '@modern-js/app-tools'
+import {
+  builderPluginAdapterBasic,
+  builderPluginAdapterHooks,
+  parseRspackConfig as parseToRsbuildConfig,
+} from '@modern-js/app-tools/builder'
+import { createStorybookOptions } from '@modern-js/plugin-v2/cli'
+// TODO: better import from `@modern-js/app-tools/builder`
+import { mergeRsbuildConfig } from '@rsbuild/core'
+import type { StorybookConfigRsbuild } from 'storybook-builder-rsbuild'
+
+const MODERN_META_NAME = 'modern-js'
+const MODERN_CONFIG_FILE = 'modern.config.ts'
+
+/**
+ * Storybook addon for Modern.js, only support bundler `rspack`.
+ */
+export const rsbuildFinal: StorybookConfigRsbuild['rsbuildFinal'] = async (
+  config,
+) => {
```

Comment:
No options for addon as of now, do you think we can simply add at least one, for example, `modernConfigFilePath` (currently config file is hard coded to `MODERN_CONFIG_FILE`).